From 8aa29f3ef365c347cb97a68f1013a8d744ee0604 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 2 Jan 2019 21:38:48 +0000
Subject: [PATCH 19/20] venv: Revert a change from
 https://github.com/python/cpython/commit/1c3de541e64f75046b20cdd27bada1557e550bcd
 https://bugs.python.org/issue35644

If copier(context.executable, path) is not also called on Windows (as it was in Python 3.7.1) then the venv\Scripts dir will not contain a Python exe
---
 Lib/venv/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Lib/venv/__init__.py b/Lib/venv/__init__.py
index 4a49b240b8..edb4e2266b 100644
--- a/Lib/venv/__init__.py
+++ b/Lib/venv/__init__.py
@@ -211,9 +211,9 @@ class EnvBuilder:
         binpath = context.bin_path
         path = context.env_exe
         copier = self.symlink_or_copy
+        copier(context.executable, path)
         dirname = context.python_dir
         if os.name != 'nt':
-            copier(context.executable, path)
             if not os.path.islink(path):
                 os.chmod(path, 0o755)
             for suffix in ('python', 'python3'):
-- 
2.21.0.windows.1

